subsystems/sidecar/index.js

```diff
@@ -789,7 +789,7 @@ class Sidecar extends ReadyResource {
       await appBundle.calibrate()
     } catch (error) {
       if (error.code === 'DECODING_ERROR') {
-        const err = new ERR_PERMISSION_REQUIRED('Permission required to run key', state.key)
+        const err = new ERR_PERMISSION_REQUIRED('Permission required to run key', state.key, true)
```

Comment:
can we make ERR_PERMISSION_REQUIRED so its (msg, info), so we can do `new ERR_PERMISSION_REQUIRED('Permission required to run key', { key: state.key, encrypted: true })`

also for consisitency-sake would you mind making this block be } catch (err) { } and then just `app.report({ err: new ERR_PERMISSION_REQUIRED(...) })`